web-api/src/main/java/com/graphhopper/util/exceptions/DetailedRuntimeException.java

```diff
@@ -24,6 +24,8 @@
  */
 public class DetailedRuntimeException extends RuntimeException implements GHException {
 
+    private static final long serialVersionUID = 1L;
```

Comment:
@otbutz Is there any specific reason you added this? From what I understand this is recommended for all classes implementing `Serializable`, and since we extend from `Throwable` here we do implement `Serializable`. Then again I don't see how we would ever serialize an exception, let alone run into a problem because we serialize an exception and it doesn't have a `serialVersionUID`?!

Reply:
> @otbutz Is there any specific reason you added this?

https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/io/Serializable.html

> it is *strongly recommended* that all serializable classes explicitly declare serialVersionUID values, since the default serialVersionUID computation is highly sensitive to class details that may vary depending on compiler implementations, and can thus result in unexpected InvalidClassExceptions during deserialization. 

> Then again I don't see how we would ever serialize an exception

We don't but that doesn't imply that a user of the grahhopper lib won't :)